tailwind.config.mjs

```diff
@@ -84,6 +86,7 @@ export default {
       zIndex: {
         1: '1',
         2: '2',
+        max: '2147483647',
```

Comment:
😬 - Still, this is the same as it was before.

Reply:
The possible bright side is that it will be much more in our face and we can elect to do something about it after we've migrated all of the CSS (i.e. we'll be able to see where it's being used and potentially move away from it).